tests/test_types.py

```diff
@@ -173,3 +173,61 @@ def test_list_errors(self):
         err = config_suite.errors[0]
         self.assertIsInstance(err, configsuite.MissingKeyError)
         self.assertEqual(("playgrounds", 0), err.key_path)
+
+    def test_date(self):
+        for date in [
```

Comment:
I would prefer to have the cases in a separate list and then iterate over that one. Makes it more readable than the multi-line loop?